server/bin/cmv

```diff
@@ -0,0 +1,446 @@
+#!/usr/bin/env ruby
```

Comment:
Could you add a little documentation here at the top just to clarify what this is for anyone who looks in future, cmv could be a bit ambiguous.